tests/process_monitor/test_comm_from_server.py

```diff
@@ -664,16 +664,101 @@ def test_MantarrayProcessesMonitor__check_and_handle_server_to_main_queue__handl
     monitor_thread, *_ = test_monitor(test_process_manager)
     server_to_main_queue = test_process_manager.queue_container.from_flask
 
-    okc_process = test_process_manager.instrument_comm_process
     fw_process = test_process_manager.file_writer_process
     da_process = test_process_manager.data_analyzer_process
-    expected_okc_item = "item 1"
-    expected_fw_item = "item 2"
-    expected_da_item = "item 3"
+    okc_process = test_process_manager.instrument_comm_process
+    expected_okc_item = {
+        "board_0": {
+            "outgoing_data": [
+                {
+                    "communication_type": "log",
+                    "log_level": 20,
+                    "message": "File Writer Process beginning teardown at 2023-02-06 23:25:38.365941",
```

Comment:
OkComm wouldn't log anything about File Writer beginning teardown, it would only log things related to itself